Gemfile

```diff
@@ -22,7 +22,7 @@ end
 # as a dependency of govuk_content_models
 gem 'mongo', '>= 1.7.1'
 
-gem 'gds-sso', '10.0.0'
+gem 'gds-sso', '11.0.0'
```

Comment:
I'm not 100% sure, because this is Mongo backed. I'd have to do some testing to be sure.

I think Mongoid allows you to use fields that haven't been defined in mass-assignment operations and just adds them to the model.  Having said that, even if it does work this way it would be much better for this to be explicit, and to add the field to content models.